addOns/custompayloads/src/main/resources/org/zaproxy/zap/extension/custompayloads/resources/Messages.properties

```diff
@@ -21,3 +21,10 @@ custompayloads.options.dialog.remove.text = Are you sure you want to remove this
 custompayloads.options.dialog.remove.button.confirm = Remove
 custompayloads.options.dialog.remove.button.cancel = Cancel
 custompayloads.options.dialog.remove.label = Do not show this message again
+
+custompayloads.options.dialog.addMultiplePayload.selectFile.button.name = Select File
+custompayloads.options.dialog.addMultiplePayload.addPayload.button.name = Add Multiple Payloads
+custompayloads.options.dialog.addMultiplePayload.duplicates.checkbox.label = Prevent Duplicates
+custompayloads.options.dialog.addMultiplePayload.add.title = Add
+custompayloads.options.dialog.addMultiplePayload.cancel.title = Cancel
+custompayloads.options.dialog.addMultiplePayload.error.title = Invalid File
```

Comment:
`Error Importing File` or `Error Adding Payloads` would be better, the file might be valid.